Assets/00_Content/Scripts/Vikings/Viking.cs

```diff
@@ -27,21 +42,38 @@ public class Viking : Interactable {
 		public int CurrentDesireIndex { get; set; }
 		public int QueuePosition { get; set; }
 
+		public bool IsSeated { get; set; }
```

Comment:
I kept this as CurrentChair behaves quite weird as it actually gets set before the viking gets seated. This could get looked at in the future, but will keep it like this for now.

Reply:
The problem is that this is set at the same time as currentchair. There is no difference as to when this is set compared to currentchair